pkg/route/api/validation/validation.go

```diff
@@ -300,9 +301,14 @@ func validateCertificatePEM(certPEM string, options *x509.VerifyOptions) (*x509.
 	}
```

Comment:
We are appending only the certs from the PEM to the cert pool. 
As re: router should be fine - only cert and key blocks would be used.